components/notion/sources/updated-page/updated-page.mjs

```diff
@@ -0,0 +1,53 @@
+import notion from "../../notion.app.mjs";
+import base from "../common/base.mjs";
+import constants from "../common/constants.mjs";
+
+export default {
+  ...base,
+  key: "notion-udpated-page",
+  name: "Updated Page", /* eslint-disable-line pipedream/source-name */
```

Comment:
Just checking... Some specific reason for this disabled lint?

Reply:
> Source names should start with "New". See https://pipedream.com/docs/components/guidelines/#source-nameeslint(pipedream/source-name)

Yeah, the lint requires that the name begins with **New**

Reply:
Got it!